### PR TITLE
Add cloudwatch_* options.

### PR DESCRIPTION
**Pull request by jotes**
Hi @Miserlou, could you review this patch? If something is unclear, please feel free to ask.

### CODE REVIEW

File: zappa/zappa.py [hunk 1]
```diff
@@ -814,27 +816,76 @@ def deploy_api_gateway(self, api_id, stage_name, stage_description="", descripti
...
+            self.apigateway_client.delete_rest_api(
+                    restApiId=api['id']
             )
 
+    def update_stage_config(self, project_name, stage_name, cloudwatch_log_level, cloudwatch_data_trace,
+        cloudwatch_metrics_enabled):
+        """
+        Update settings on staging.
+        """
+        print("Update configuration on staging...")
```
Comment (Miserlou) [inline zappa/zappa.py]: This should be 'on %s' % stage_name ?

File: zappa/zappa.py [hunk 2]
```diff
@@ -225,6 +225,7 @@ class Zappa(object):
     assume_policy = ASSUME_POLICY
     attach_policy = ATTACH_POLICY
     aws_region = 'us-east-1'
+    log_levels = ['OFF', 'ERROR', 'INFO']
```
Comment (Miserlou) [inline zappa/zappa.py]: I'd change this to cloudwatch_log_levels to avoid confusion with Python log levels which we also use

**Comment by Miserlou** [comment]
Looks okay, but I think the print("Update configuration on staging...") is wrong..

**Comment by jotes** [comment]
@Miserlou Hi, could you look again?

**Comment by Miserlou** [comment]
👍

**Pull request merged**